x/profile/internal/types/keys.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import "regexp"
+
+const (
+	ModuleName = "profile"
+	RouterKey  = ModuleName
+	StoreKey   = ModuleName
+
+	MaxNameSurnameLength = 500
+	MaxMonikerLength     = 30
+	MaxBioLength         = 1000
+
+	ActionCreateProfile = "create_profile"
+	ActionEditProfile   = "edit_profile"
+	ActionDeleteProfile = "delete_profile"
+
+	//Queries
+	QuerierRoute  = ModuleName
+	QueryProfile  = "profile"
+	QueryProfiles = "profiles"
+)
+
+var (
+	TxHashRegEx = regexp.MustCompile("^[a-fA-F0-9]{64}$")
+
+	ProfileStorePrefix = []byte("profile")
+)
+
+// ProfileStoreKey turns a moniker to a key used to store a profile into the profiles store
+func ProfileStoreKey(moniker string) []byte {
```

Comment:
Why have you decided to store based on the moniker and not the address of a user?

Reply:
I was thinking that could be useful for developers having the possibility to query an account from the moniker, but honestly I didn't think that we could achieve the same results using the address. I think we could use both of them, to give them two ways to query on them even on chain (cause probably this will be done automatically using GQL)

Reply:
@bragaz Yes please, keep both the queries: address and moniker as they will be both surely be used by people.